oid4vci/oid4vci/models/exchange.py

```diff
@@ -20,7 +20,7 @@ class Meta:
         schema_class = "OID4VCIExchangeRecordSchema"
 
     RECORD_TYPE = "oid4vci"
-    EVENT_NAMESPACE = "oid4vci"
+    # EVENT_NAMESPACE = "oid4vci"
```

Comment:
Acapy webhook is looking for `EVENT_NAMESPACE: str = "acapy::record"` to send the message, by commenting this out, it will fallback to the one set inside `base_record.py`